blockprotocol/blockprotocol
# Add searchbar for mobile

## What this Does

PR #125 added search to docs pages. However, the mobile version of the searchbar for the `docs` and `spec` pages wasn't implemented immediately. This PR implements those, and makes a few other changes in styles wherever improvements could be made.

Changes are documented in GH comments below.

## How to test this

1. Checkout this branch locally
2. Check both the mobile and desktop versions of the searchbar and test for bugs.

> Demo Note: The pages take time to build for me locally while I'm recording the screen.

![chrome_i8qqoJAB64](https://user-images.githubusercontent.com/22196279/152127477-8cbd814b-aa3c-41a0-8bea-b9ea096b3648.gif)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201706447225517
